libbpf-tools/profile.c

```diff
@@ -595,6 +631,10 @@ int main(int argc, char **argv)
 				env.perf_max_stack_depth * sizeof(unsigned long));
 	bpf_map__set_max_entries(obj->maps.stackmap, env.stack_storage_size);
 
+	err = set_pidns(obj);
+	if (err)
```

Comment:
Let us do 'err && env.verbose' here. We want to print out the below information under verbose mode.

Reply:
it's modified. thank you.